presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/GeoFunctions.java

```diff
@@ -1242,4 +1291,28 @@ private static boolean envelopes(Slice left, Slice right, EnvelopesPredicate pre
     {
         boolean apply(Envelope left, Envelope right);
     }
+
+    private static Iterable<Slice> getSlicesFromBlock(Type type, Block block)
```

Comment:
since this function only used for geometries, perhaps, simplify to `getGeometrySlicesFromBlock(block)`